src/lib/operators/print.cpp

```diff
@@ -60,6 +60,14 @@ std::shared_ptr<const Table> Print::_on_execute() {
     _out << "||_BEGIN|_END  |_TID  ";
   }
   _out << "|" << std::endl;
+  for (ColumnID column_id{0}; column_id < input_table_left()->column_count(); ++column_id) {
```

Comment:
The fact that you could change this without having to alter the OperatorPrintTests makes me worry about the coverage of the Print operator...

Reply:
Did I hear my name?